src/makeCalculator.js

```diff
@@ -37,7 +37,23 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    add: (a, b) => a + b,
+    subtract: (a, b) => a - b,
+    multiply: (a, b) => a * b,
+    divide: (a, b) => a / b,
+    operate(operation, number) {
+      this.result = operation(this.result, number);
+
+      return this;
+    },
```

Comment:
Arrow functions are great, but let's use one approach for creating methods, why some of methods created by arrow functions and some by function declaration?